geomesa-web/geomesa-web-security/src/main/scala/org/locationtech/geomesa/web/security/GMSecuredDataFactory.scala

```diff
@@ -0,0 +1,91 @@
+package org.locationtech.geomesa.web.security
+
+import com.google.common.collect.Maps
+import org.apache.accumulo.core.security.{Authorizations, ColumnVisibility, VisibilityEvaluator}
+import org.geoserver.security.WrapperPolicy
+import org.geoserver.security.decorators.{DecoratingFeatureSource, DefaultSecureDataFactory}
```

Comment:
This will have to be submitted to geoserver down the road.  We need it sooner than that though.